src/components/content/Content.js

```diff
@@ -2,69 +2,94 @@ import Greetings from '../greetings/index';
+import Button from '../button';
+import Show from '../show';
+import Location from '../location';
 
-const Error = (props) => {
-  console.log(props);
-  const date = props.date ? props.date.toLocaleString() : '';
-  const style = {backgroundColor: 'red', color: props.color || 'white'};
-
-  return <mark style={style}>There is an error: {props.text} {date}</mark>
-};
-
-const list = [
-  { 
-    firstName: 'Erik',
-    lastName: 'Cartman',
-    age: 12
-  },
-  {
-    firstName: 'Kyle',
-    lastName: 'Broflovsky',
-    age: 12
-  },
-  {
-    firstName: 'Stan',
-    lastName: 'Marsch',
-    age: 12
-  },
-];
+import './content.scss';
 
+// eslint-disable-next-line no-undef
 class Content extends Component {
-  state = { users: [] };  
+  state = {
+    users: [],
+    classN: false,
+    btnText: 'show',
+    rectClass: 'hidden',
+    userPosts: [],
+    latitude: '',
+    longtitude: ''
+  };
 
   componentDidMount() {
     fetch('https://jsonplaceholder.typicode.com/users')
       .then(resp => resp.json())
       .then(users => this.setState({ users }));
+
+    navigator.geolocation.getCurrentPosition((pos) => {
+      this.setState({
+        latitude: pos.coords.latitude,
+        longtitude: pos.coords.longitude
+      })
+    });
+  }
+
+  toggleClass = () => {
+    const { classN } = this.state;
+    this.setState(classN ? { classN: false } : { classN: true });
+  }
+
+  toggleView = () => {
+    const { rectClass } = this.state;
+
+    this.setState(
+      rectClass === 'hidden' ? {
+        rectClass: 'shown',
+        btnText: 'hide'
+      } : {
+        rectClass: 'hidden',
+        btnText: 'show'
+      }
+    );
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  showUserInfo = (user) => {
+    fetch(`https://jsonplaceholder.typicode.com/posts?userId=${user.id}`)
+      .then(resp => resp.json())
+      .then(posts => this.setState({ userPosts: posts }));
   }
 
-  showUserInfo(user) {
-    alert(user.name);
+  closeInfo = () => {
+    this.setState({ userPosts: [] });
   }
 
   render() {
-    const { users } = this.state;
+    const {
+      users,
+      classN,
+      btnText,
+      rectClass,
+      userPosts,
+      latitude,
+      longtitude
+    } = this.state;
 
     return (
       <div className="content">
         <h2> Content </h2>
         <Greetings name="Alex" />
         <Numbers from={1} to={8} even />
         <UsersTable users={list} />
-        {/* <ul>
-          {
-            users.map(({ name, id, username }) => 
-            <li key={id} onClick={() => this.showUserName(username)}>{name}</li>)
-          }
-        </ul> */}
         <Counter />
-        <List items={users} clickHandler={this.showUserInfo} />
-      </div>  
-    )
+        <Button handleClick={this.toggleClass} btnClassName={classN} />
+        <Show handleClick={this.toggleView} text={btnText} btnClassName={rectClass} />
+        <Location latitude={latitude} longtitude={longtitude} />
```

Comment:
Location should work with `navigator.geolocation`, it should be independent component